runtime/test/manifest-test.js

```diff
@@ -22,6 +22,26 @@ async function assertRecipeParses(input, result) {
 }
 
 describe('manifest', function() {
+  it('duplicate consume slot', async () => {
+    let manifest = await Manifest.parse(`
+      particle SomeParticle in 'some-particle.js'
+        work()
+        consume slotA
+        consume slotB
+      particle SomeParticle1 in 'some-particle.js'
+        rest()
+        consume slotC
+
+      recipe
+        SomeParticle
+          consume slotA
+        SomeParticle1
+          consume slotC
+    `);
+    let recipe = manifest.recipes[0];
+    assert(recipe);
+  });
```

Comment:
What's this test for?

Reply:
I updated the test to really reflect what it tests, but basically before the current change in the manifest.js file parsing this recipe would cause an assert to fire.